src/tyr/matrix_serializer.cc

```diff
@@ -43,6 +43,32 @@ json::ArrayPtr serialize_distance(const valhalla::Matrix& matrix,
   }
   return distance;
 }
+
+json::ArrayPtr serialize_shape(const valhalla::Matrix& matrix,
```

Comment:
I tried to do this whole thing a bit nicer in the serializer, but the fact that geojson & polyline are different types (map vs string), I couldn't come up with smth nicer without a lot more changes. honestly this whole serializer needs a complete overhaul. it's just hard to come up with terse/elegant code for as many options as we provide for the response :)